File: frontend/src/app/business/dtos/literature-category.response.ts
export class LiteratureCategoryResponse {
  private _id: number;
  private _categoryName: string;

  public constructor() {
  }

  get id(): number {
    return this._id;
  }

  set id(value: number) {
    this._id = value;
  }

  get categoryName(): string {
    return this._categoryName;
  }

  set categoryName(value: string) {
    this._categoryName = value;
  }
}

File: backend/src/main/java/com/mendes/library/model/DTO/LiteratureCategoryDTO/LiteratureCategoryResponse.java
package com.mendes.library.model.DTO.LiteratureCategoryDTO;


import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
public class LiteratureCategoryResponse {

    private Long id;
    private String categoryName;

}
